conf/zeppelin-env.sh.template

```diff
@@ -23,6 +23,9 @@
 # export ZEPPELIN_INTP_MEM       		# zeppelin interpreter process jvm mem options. Default -Xms1024m -Xmx1024m -XX:MaxPermSize=512m
 # export ZEPPELIN_INTP_JAVA_OPTS 		# zeppelin interpreter process jvm options.
 # export ZEPPELIN_SSL_PORT       		# ssl port (used when ssl environment variable is set to true)
+# export ZEPPELIN_JMX_ENABLE        # Enable JMX feature by defining it like "true"
+# export ZEPPELIN_JMX_PORT          # Port number which JMX uses. Default: "9996"
+# export ZEPPELIN_JMX_OVERRIDE      # Override all of JMX options. This will replace any default setting including ZEPPELIN_JMX_PORT
```

Comment:
align space?

Reply:
Huh... existing ones were tabs and github seems to replaces a tab to 8 spaces

Reply:
Fixed